### DISCUSSION TITLE
Adding more special characters

### DISCUSSION THREAD

**Discussion by ghost** [monkeytypegame/monkeytype#2087] [Ideas]
Key/symbol | Explanation
-- | --
~ | Tilde.
` | Acute, back quote
! | Exclamation mark
@ |  at
'#' | hash.
$ | Dollar sign or generic currency.
% | Percent.
^ | Caret or circumflex.
& | Ampersand
'*' | Asterisk
( | Open or left parenthesis.
) | Close or right parenthesis.
'-' | Hyphen, minus, or dash.
_ | Underscore.
'+' | Plus.
= | Equal.
{ | Open brace, squiggly brackets, or curly bracket.
} | Close brace, squiggly brackets, or curly bracket.
[ | Open bracket.
] | Closed bracket.
\| | vertical bar.
\ | Backslash
/ | Forward slash
: | Colon.
; | Semicolon.
" | Quote, quotation mark, or inverted commas.
' | Apostrophe or single quote.
< | Less than or angle brackets.
'>' | Greater than or angle brackets.
, | Comma.
. | Period, dot or full stop.
? | Question mark.
|---|---

**Comment by Miodec** [comment]
ASCII and Secials funboxes do that.

**Reply by ghost** [reply]
- [ ] ASCII & Specials are nice but i feel it will be better if you can add some thing like a new mode dedicated for programmers with name "coding" for people who need speed while programming which consist of special characters,numbers,words etc... and its combination

**Reply by unrealapex** [reply]
You could type code tests  [code quotes are nice(`command line>language...>code java`)] or put those characters in a custom test and enable random word generation.
